app/models/concerns/hyku_addons/account_behavior.rb

```diff
@@ -74,8 +75,27 @@ def switch_host!(cname)
         Rails.application.routes.default_url_options[:host] = cname
         Hyrax::Engine.routes.default_url_options[:host] = cname
       end
+
+      def setup_tenant_cache
+        Rails.application.config.action_controller.perform_caching = api_cache?
+        ActionController::Base.perform_caching = api_cache?
+
+        Rails.application.config.cache_store = :redis_cache_store, { url: Redis.current.id } if api_cache?
+        Rails.cache = ActiveSupport::Cache.lookup_store(Rails.application.config.cache_store)
+      end
+
+      def disable_tenant_cache
+        Rails.application.config.action_controller.perform_caching = api_cache?
+        ActionController::Base.perform_caching = api_cache?
+
+        Rails.application.config.cache_store = :file_store, Settings.cache_filesystem_root
+        Rails.cache = ActiveSupport::Cache.lookup_store(Rails.application.config.cache_store)
+      end
+
+      def api_cache?
```

Comment:
Now that we have renamed the flipflop, we should rename this method to match, so we dont have `api_cache` and `cache_api`, which will be hard to remember which is which. Can this become `cache_api?`.